Assets/Scripts/StandardSamples/UI/Common/UIDebugLog.cs

```diff
@@ -77,7 +77,11 @@ private void Start()
             logLevelMenuItems = new List<UIDebugLogLevelMenuItem>();
             BuildLogLevelMenu();
             ignoreLogLevelChange = false;
-            LogLevelScrollView.gameObject.SetActive(false);
+
+            if (LogLevelScrollView != null)
+            { 
+                LogLevelScrollView.gameObject.SetActive(false);
```

Comment:
Would this be a good case to use `null` propagation? 
```csharp
LogLevelScrollView?.gameObject.SetActive(false);
```
I know that in certain areas of Unity, use of `null` propagation is discouraged, but I never understood why or what the borders of that recommendation are.

Reply:
Resolved as you suggested; this is a clear time to use the null propagation technique.